src/main/resources/db/changelog/changes/sql/2023-08-17-01__create_transaction_table.sql

```diff
@@ -1,18 +1,20 @@
-create table transactions
-(
-    transaction_id         varchar not null primary key,
+CREATE TYPE StatusEnum as ENUM ('CREATED', 'OPEN', 'CANCELLED', 'IN_TRANSIT', 'AWAITING_PICKUP', 'ITEM_CHECKED_OUT', 'ITEM_CHECKED_IN', 'IN_TRANSIT_TO_LENDING', 'CLOSED', 'ERROR');
```

Comment:
@gurleenkaurbp , @Vignesh-kalyanasundaram 
I think we should not write sql script directly .
instead we should use xml script . this will make code decoupled . another thing is tracking DB changes is difficult in sql script because lot of manual efforts required but with xml script Liquibase will automatically keep track of changes and we can rollback easily.  

Please suggest ... thanks.

Reply:
@SinghAdes , perfect catch. Thx.
fixed